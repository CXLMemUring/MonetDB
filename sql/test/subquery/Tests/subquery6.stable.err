stderr of test 'subquery6` in directory 'sql/test/subquery` itself:


# 11:45:43 >  
# 11:45:43 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-297142" "--port=31114"
# 11:45:43 >  

MAPI  = (monetdb) /var/tmp/mtest-297142/.s.monetdb.31114
QUERY = DELETE FROM another_t WHERE (SELECT 1 UNION SELECT 2) > 1; 
ERROR = !GDK reported error: BATsubcross: more than one match
MAPI  = (monetdb) /var/tmp/mtest-297142/.s.monetdb.31114
QUERY = DELETE FROM another_t WHERE (SELECT 1 UNION SELECT 2) > 1; 
ERROR = !GDK reported error: BATsubcross: more than one match

# 11:45:43 >  
# 11:45:43 >  "Done."
# 11:45:43 >  

