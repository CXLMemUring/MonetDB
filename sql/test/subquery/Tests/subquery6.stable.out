stdout of test 'subquery6` in directory 'sql/test/subquery` itself:


# 11:45:43 >  
# 11:45:43 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-297142" "--port=31114"
# 11:45:43 >  

#CREATE TABLE tbl_ProductSales (ColID int, Product_Category  varchar(64), Product_Name  varchar(64), TotalSales int); 
#INSERT INTO tbl_ProductSales VALUES (1,'Game','Mobo Game',200),(2,'Game','PKO Game',400),(3,'Fashion','Shirt',500),(4,'Fashion','Shorts',100);
[ 4	]
#CREATE TABLE another_T (col1 INT, col2 INT, col3 INT, col4 INT, col5 INT, col6 INT, col7 INT, col8 INT);
#INSERT INTO another_T VALUES (1,2,3,4,5,6,7,8), (11,22,33,44,55,66,77,88), (111,222,333,444,555,666,777,888), (1111,2222,3333,4444,5555,6666,7777,8888);
[ 4	]
#CREATE TABLE integers(i INTEGER);
#INSERT INTO integers VALUES (1), (2), (3), (NULL);
[ 4	]
#WITH customer_total_return AS
#  (SELECT 1 AS ctr_customer_sk,
#          1 AS ctr_state,
#          1 AS ctr_total_return)
#SELECT 1
#FROM customer_total_return ctr1,
#     another_T,
#     tbl_ProductSales
#WHERE ctr1.ctr_total_return >
#    (SELECT avg(ctr_total_return)*1.2
#     FROM customer_total_return ctr2
#     WHERE ctr1.ctr_state = ctr2.ctr_state)
#  AND col1 = ColID
#  AND ctr1.ctr_customer_sk = TotalSales;
% . # table_name
% single_value # name
% tinyint # type
% 1 # length
#SELECT (SELECT (SELECT SUM(col1)) IN (MAX(col2))) FROM another_t;
% .%5 # table_name
% %5 # name
% boolean # type
% 5 # length
[ false	]
#DROP TABLE tbl_ProductSales;
#DROP TABLE another_T;
#DROP TABLE integers;

# 11:45:43 >  
# 11:45:43 >  "Done."
# 11:45:43 >  

